src/store/index.js

```diff
@@ -24,12 +24,14 @@ const reducer = combineReducers({
   routing,
   browser: responsiveStateReducer,
 });
+/* eslint-disable no-underscore-dangle */
```

Comment:
I agree in principle, but in this case, it's probably something we should avoid: however, it's a 'hack' so that we can make redux dev tools work (it's the only way to hook it into the store basically).